calculator.rb

```diff
@@ -0,0 +1,90 @@
+# Calculator (Dinner) - Loraine Kanervisto
+
+# Starts with user prompts and data collection
+
+#Collects first number
+puts "You find a calculator on the ground. You pick it up. The screen says:\n
+Enter a number>>"
+first_num = gets.chomp.to_f
+
+#Checks first number
+while first_num != "0" && first_num.to_i == 0
```

Comment:
These conditions are a little difficult for me parse. If you're always converting the result of `gets.chomp` to a float, `first_num != "0"` won't ever evaluate to true. The second condition totally works if someone types in a word (also it works without converting the float to an integer), but what happens when someone types in 0?